charts/hlf-k8s/README.md

```diff
@@ -125,18 +127,21 @@ Install a chaincode on a peer using the following values.
 On a peer:
 
 ```yaml
+chaincodes:
+  - name: mycc
+    version: "1.0"
+    address: "chaincode-org-0-substra-chaincode-chaincode.org-0"
+    port: "7052"
+    image:
+      repository: substrafoundation/substra-chaincode
+      tag: 0.1.0
+      pullPolicy: IfNotPresent
+
 appChannels:
   - channelName: mychannel
     chaincodes:
       - name: mycc
-        version: "1.0"
```

Comment:
isn't the version necessary for the instantiation in the channel ? I'm not completely sure how it works with the lifecycle management introduced with hlf 2.x but on 1.4 you needed to pass the chaincode version during the `instatiate` command.

Reply:
The version number is still specified, although the field has been moved from `appChannels[].chaincodes[].version` to `chaincodes[].version`

Reply:
yes but that's not what I meant: in 1.4 for example you could install version x, y and z and choose to instantiate the version x in the channel A and y in the channel B. It doesn't seems possible anymore with this structure.

Reply:
Good point! 

We should be able to replicate this behavior by creating multiple chaincodes 

```
chaincodes:
- name: mycc-v1
  version: 1.0
- name: mycc-v2
  version: 2.0

appChannels:
- name: mychannel
  chaincodes: 
  - name: mycc-v1
- name: yourchannel
  chaincodes: 
  - name: mycc-v2
```

Although we haven't tested it yet.

Reply:
As I understand here: https://hyperledger-fabric.readthedocs.io/en/release-2.2/chaincode_lifecycle.html#the-channel-does-not-agree-on-a-chaincode-definition it seems the chaincode commit phase still requires a pair chaincode name + version. I think you will need to put the version in both chaincodes[] (for chaincode packaging and installation) and appChannels[].chaincodes[] (for approving the definition and commiting to the channel).

Reply:
Yes, it's done [here](https://github.com/SubstraFoundation/hlf-k8s/pull/102/files#diff-93117691d56deeeb90f61349a7817cb5217d0b31080c427fecb3c683e718014dR342-R346) with chaincode properties fetch [here](https://github.com/SubstraFoundation/hlf-k8s/pull/102/files#diff-93117691d56deeeb90f61349a7817cb5217d0b31080c427fecb3c683e718014dR292-R302) @ClementGautier 

For information we use this behavior for the chaincode definition : https://hyperledger-fabric.readthedocs.io/en/release-2.2/chaincode_lifecycle.html#organizations-install-different-chaincode-packages

Reply:
@Kelvin-M exactly, what I'm saying is that you shouldn't fetch the version in the chaincodes[] array based on the name of the chaincode name because that basicly forbid you to have multiple version of a same chaincode and make you use unecessary workaround like the one suggested by @AurelienGasser (putting the version in the name itself). A chaincode upgrade should be as easy as adding a new version of a chaincode in the chaincodes[] list and changing the one used in a channel by changing it's version in the appChannel.chaincode.version in my opinion. Let's discuss this on vocal this afternoon?